commons-redis/src/main/scala/com/avsystem/commons/redis/RedisNodeClient.scala

```diff
@@ -78,7 +109,8 @@ final class RedisNodeClient(
   private[redis] def nodeRemoved(): Unit = {
     val cause = new NodeRemovedException(address)
     failure = cause.opt
-    connections.foreach(_ ! Close(cause))
+    connections.foreach(_ ! RedisConnectionActor.Close(cause, stop = true))
```

Comment:
Both `Close` classes seem to always be used with `stop = true`

Reply:
I'd like to leave a possibility to close connection without stopping the actor, even though it's not used now.